enhancements/monitoring/scrape-profiles.md

```diff
@@ -0,0 +1,274 @@
+---
+title: scrape-profiles
+authors:
+  - @JoaoBraveCoding
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - @openshift/openshift-team-monitoring
+approvers:
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: 2022-12-06
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/MON-2483
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Scrape profiles
+
+## Summary
+
+The core OpenShift components ship a large number of metrics. A 4.12-nightly
+cluster on AWS currently produces around 350K series by default, and enabling
+additional add-ons increases that number. Users have repeatedly asked for a supported
+method of making Prometheus consume less memory, either by increasing the scraping
+interval or by scraping fewer targets -- for example, modifying the ServiceMonitors
+to drop metrics undesired to some users.
+
+These modifications are currently not possible, because the service monitors deployed to OCP
+are managed by operators and cannot be modified. This proposal outlines a solution for allowing
+users to set a level of scraping aligned with their needs.
```

Comment:
```suggestion
These modifications are currently not possible, because the OpenShift operators manage their service/pod monitors (either directly or via the cluster version operator) and any manual modification would be reverted. This proposal outlines a solution for allowing users to set a level of scraping aligned with their needs.
```